src/controllers/imgurController.js

```diff
@@ -0,0 +1,83 @@
+const { ImgurClient } = require('imgur');
+const User = require('../models/User');
+
+const imgurClient = new ImgurClient({
+  clientId: process.env.IMGUR_CLIENT_ID,
+  clientSecret: process.env.IMGUR_CLIENT_SECRET,
+  refreshToken: process.env.IMGUR_REFRESH_TOKEN,
+});
+
+const renderUploadPage = (req, res, message, imageUrl) => {
```

Comment:
All controller functions can only take two values, `req`, and `res`. Otherwise, Express won't be able to handle it. If we need to pass in `message`, and `imageUrl` we should be passing it in via params, query parameters, or a request body.

```suggestion
const renderUploadPage = (req, res) => {
```